modules/host.go

```diff
@@ -53,6 +48,26 @@ var (
 	// RPCDownload is the specifier for downloading a file from a host.
 	RPCDownload = types.Specifier{'D', 'o', 'w', 'n', 'l', 'o', 'a', 'd', 2}
 
+	// SectorSize defines how large a sector should be in bytes. The sector
+	// size needs to be a power of two to be compatible with package
+	// merkletree. 4MB has been chosen for the live network because large
+	// sectors significantly reduce the tracking overhead experienced by the
+	// renter and the host.
+	SectorSize = func() uint64 {
+		if build.Release == "dev" {
+			return 1 << 20 // 1 MiB
+		}
+		if build.Release == "standard" {
+			return 1 << 22 // 4 MiB
+		}
+		if build.Release == "testing" {
+			return 1 << 12 // 4 KiB
+		}
+		panic("unrecognized release constant in host - sectorSize")
```

Comment:
did we decide that there was a better way to do this? I forget. Oh, I think we were planning on adding type safety to catch misspellings at compile time.